pkg/apis/componentconfig/v1alpha1/zz_generated.conversion.go

```diff
@@ -502,7 +435,8 @@ func autoConvert_componentconfig_CloudControllerManagerConfiguration_To_v1alpha1
 	if err := Convert_componentconfig_CloudProviderConfiguration_To_v1alpha1_CloudProviderConfiguration(&in.CloudProvider, &out.CloudProvider, s); err != nil {
 		return err
 	}
-	if err := Convert_componentconfig_DebuggingConfiguration_To_v1alpha1_DebuggingConfiguration(&in.Debugging, &out.Debugging, s); err != nil {
+	// TODO: Inefficient conversion - can we improve it?
+	if err := s.Convert(&in.Debugging, &out.Debugging, 0); err != nil {
```

Comment:
likely need to declare a dependency, like https://github.com/kubernetes/kubernetes/pull/66030/files